src/utils/translateWhiskyNameToJapenese.ts

```
@@ -0,0 +1,115 @@
+    if (match && match[1]) return match[1]; // 숫자 그룹만 반환
+  }
+
+  // 숫자만 있는 경우 (예: 12)
+  const numberMatch = input.match(/\d+/);
+  if (numberMatch) return numberMatch[0];
+
+  return "";
+}
+
+// 검색 함수 수정
+export function translateWhiskyNameToJapenese(input: string): {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)